docs/torch/ray_usage.rst

```diff
@@ -0,0 +1,297 @@
+****************************
+Node Classification with GAT
+****************************
+
+In this guide we use a pre-built `Graph Attention Network(GAT) <https://arxiv.org/abs/1710.10903>`_ model
```

Comment:
Please, reference the original documentation for node classification and avoid copy-pasting. You can also use some other model to show how to work with ray.

Reply:
I'll reference original documentation and update the title, I should keep the GAT model though since we are going to replace old documentation in the near future anyway - it is a lot of work to get a new model in with Ray Data.